test/testHeadless.m

```diff
@@ -35,6 +36,7 @@
         makeTravisReport(status);
     end
 
+    restoreGlobalState(state);
```

Comment:
(I also got confused with that name in the beginning, why they didn't just implement a `try-finally` still boggles me)